src/LMTR_alg.jl

```diff
@@ -131,7 +131,7 @@ function LMTR(
   Jt_Fk = similar(∇fk)   # temporary storage
 
   σmax = opnorm(Jk)
-  νInv = (1 + θ) * σmax^2  # ‖J'J‖ = ‖J‖²
+  νInv = σmax^2/θ # ‖J'J‖ = ‖J‖²
```

Comment:
```suggestion
  νInv = σmax^2 / θ # ‖J'J‖ = ‖J‖²
```
It's good practice to add a blank line between variables for readability.